.github/workflows/linux.yml

```diff
@@ -362,3 +366,51 @@ jobs:
         rm -rf build-install
         ./vendored-meson/meson/meson.py install -C build --destdir ../build-install --tags=runtime,python-runtime,devel
         python tools/check_installed_files.py $(find ./build-install -path '*/site-packages/numpy') --no-tests
+
+
+
+  Linux_Python_311_32bit_full_with_asserts:
+    name: i686, cp311, full, with asserts
+    needs: [smoke_test]
+    runs-on: ubuntu-latest
+    container:
+       # There are few options for i686 images at https://quay.io/organization/pypa,
+       # use the glibc2.17 one (manylinux2014)
+      image: quay.io/pypa/manylinux2014_i686
+
+    steps:
+    - name: Checkout and initialize submodules
+      # actions/checkout doesn't work in a container image
+      run: |
+        git config --global --add safe.directory $PWD
+        if [ $GITHUB_EVENT_NAME != pull_request ]; then
+            git clone --recursive --branch=$GITHUB_REF_NAME https://github.com/${GITHUB_REPOSITORY}.git $GITHUB_WORKSPACE
+            git reset --hard $GITHUB_SHA
+        else
+            git clone --recursive https://github.com/${GITHUB_REPOSITORY}.git $GITHUB_WORKSPACE
+            git fetch origin $GITHUB_REF:my_ref_name
+            git checkout $GITHUB_BASE_REF
+            git -c user.email="you@example.com" merge --no-commit my_ref_name
+        fi
+        git submodule update --init --recursive
+
+    - name: build
+      run: |
+        python3.11 -m venv venv
+        source venv/bin/activate
+        pip install --upgrade pip
+        pip install -r requirements/ci32_requirements.txt
+        pip install -r requirements/test_requirements.txt
+        
+        export CFLAGS="-msse2 -std=c99 -UNDEBUG"
+        export F77=gfortran-5
+        export F90=gfortran-5
+        spin config-openblas --with-scipy-openblas=32
+        export PKG_CONFIG_PATH=/numpy/.openblas
+        python -m pip install . -v
```

Comment:
Same as in my other review, I suggest using `-Dallow-noblas=false` here.

Reply:
I like using `pip install` in a couple of jobs, since (a) it uses fewer dependencies and (b) tests that using a scipy-openblas wheel without vendoring it in works as it should.